src/java.base/share/classes/java/nio/channels/FileChannel.java

```diff
@@ -997,62 +997,61 @@ public abstract MappedByteBuffer map(MapMode mode, long position, long size)
-     *         {@code offset + size} overflows the range of {@code long}.
+     * @return  A new mapped memory segment.
      *
-     * @throws IllegalStateException
-     *         If the {@code session} is not
-     *         {@linkplain MemorySession#isAlive() alive}.
+     * @throws  IllegalArgumentException
+     *          If {@code offset < 0}, {@code size < 0} or
+     *          {@code offset + size} overflows the range of {@code long}.
      *
-     * @throws WrongThreadException
-     *         If this method is called from a thread other than the thread
-     *         {@linkplain MemorySession#ownerThread() owning} the
-     *         {@code session}.
+     * @throws  IllegalStateException
+     *          If the {@code session} is not
+     *          {@linkplain MemorySession#isAlive() alive}.
      *
-     * @throws NonReadableChannelException
-     *         If the {@code mode} is {@link MapMode#READ_ONLY READ_ONLY} or
-     *         an implementation specific map mode requiring read access,
-     *         but this channel was not opened for reading.
+     * @throws  WrongThreadException
+     *          If this method is called from a thread other than the thread
+     *          {@linkplain MemorySession#ownerThread() owning} the
+     *          {@code session}.
      *
-     * @throws NonWritableChannelException
-     *         If the {@code mode} is {@link MapMode#READ_WRITE READ_WRITE},
-     *         {@link MapMode#PRIVATE PRIVATE} or an implementation specific
-     *         map mode requiring write access, but this channel was not
-     *         opened for both reading and writing.
+     * @throws  NonReadableChannelException
+     *          If the {@code mode} is {@link MapMode#READ_ONLY READ_ONLY} or
+     *          an implementation specific map mode requiring read access,
+     *          but this channel was not opened for reading.
      *
-     * @throws IOException
-     *         If some other I/O error occurs.
+     * @throws  NonWritableChannelException
+     *          If the {@code mode} is {@link MapMode#READ_WRITE READ_WRITE},
+     *          {@link MapMode#PRIVATE PRIVATE} or an implementation specific
+     *          map mode requiring write access, but this channel was not
+     *          opened for both reading and writing.
      *
-     * @throws UnsupportedOperationException
-     *         If an unsupported map mode is specified.
+     * @throws  IOException
+     *          If some other I/O error occurs.
+     *
+     * @throws  UnsupportedOperationException
+     *          If an unsupported map mode is specified.
      *
-     * @since 19
+     * @since   19
      */
     @PreviewFeature(feature=PreviewFeature.Feature.FOREIGN)
     public MemorySegment map(MapMode mode, long offset, long size,
-                                      MemorySession session)
-            throws IOException
-    {
+                             MemorySession session)
+        throws IOException {
```

Comment:
Thanks for fixing up the formatting issues. For this one then you could put MemorySession parameter on the previous line if you want. Also have "throws IOException {" indent by 4 looks a bit odd here. It can be indented by 8 or put the "{" on the next line like the other methods in this area.

Reply:
Fixed by ab19acc6174636475ab15ef7ff403782f5493926.